src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/commands.py

```diff
@@ -33,12 +33,16 @@
         c.command('failover', 'failover_replication_link')
         c.command('force-failover', 'failover_replication_link_allow_data_loss')
 
-    # Data Warehouse will not be included in the first batch of GA commands
-    # with s.group('sql db data-warehouse') as c:
-    #     c.command('pause', 'pause_data_warehouse')
-    #     c.command('resume', 'resume_data_warehouse')
+    with s.group('sql dw') as c:
+        c.custom_command('create', 'dw_create')
+        c.command('show', 'get')
```

Comment:
--param "a b c" would be incorrect unless the parameter value is supposed to be the single string "a b c". If you can specify multiple things to expand, then nargs='+' would be the idiomatic was to accomplish this. You'd have to transform this list of values into the format required by the query string (either in a validator or custom command).